helm/cluster-openstack/templates/machine_deployment.yaml

```diff
@@ -26,6 +27,6 @@ spec:
       infrastructureRef:
         apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
         kind: OpenStackMachineTemplate
-        name: {{ include "resource.default.name" $ }}-{{ .class }}-{{ include "osmtRevisionByClass" $ }}
+        name: {{ include "resource.default.name" $ }}-{{ .class }}-{{ include "osmtRevisionByClass" (merge . $) }}
```

Comment:
why this is needed after this change?

Reply:
It's equivalent to this line from the deleted file https://github.com/giantswarm/cluster-openstack/blob/92d148917ec39755d285cb4652ce6c934e433dea/helm/cluster-openstack/templates/list.yaml#L13